test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/LocalSetup.md

```diff
@@ -0,0 +1,35 @@
+﻿## Software Installation
+
+Function apps for the test run in docker containers. Please install docker on the system.
+
+## Additional Resources
+
+For the tests to run end-to-end we need to precreate the following resources - 
+
+Azure Resources: Create the following the azure resources in the same subscription
+1. Resource Group: Name can be configured as RESOURCE_GROUP in Constants file
```

Comment:
how would it be to specify the rg and ns name instead of asking the user to configure in the code ?

Reply:
Good Idea! We can take this up in a separate improvement PR.

Reply:
Sure make sense, please update the scope in that PR and share the link in the comment so that it should not be missed

Reply:
Sure. Will add it when I pick this item.